wbsv/archiver.py

```diff
@@ -1,9 +1,11 @@
-import sys
 from argparse import Namespace
 from typing import Literal, Tuple, Union
 
+import requests
```

Comment:
```suggestion
```

Reply:
It is necessary to be able to catch `requests.exceptions.TooManyRedirects` which can be raised inside the save() method

Reply:
sorry, it is ok.